### PR TITLE
Update main.json

### PR DESCRIPTION
**Pull request by alternadom**
Adding future translation possibilities

### CODE REVIEW

File: lang/main.json [hunk 1]
```diff
@@ -864,6 +864,7 @@
         "header": "Help center"
     },
     "lobby": {
+        "knockingParticipantList" : "Knocking participant list",
```
Comment (damencho) [inline lang/main.json:867]: Where is this used? I don't see it in the code.

Comment (damencho) [reply lang/main.json:867]: I see it is not translated string from code.

Comment (alternadom) [reply lang/main.json:867]: In fact it is part of a larger plan ^^ I make another pull request in order to replace the string by a var .. We are french and "Knocking" is just a joke for us 'knock, knock' :)

Comment (alternadom) [reply lang/main.json:867]: Sorry, the "larger plan" was not deployed.. #7249

**Pull request merged**